samples/ChatbotExamplev2/Bots/PersonalizerChatbot.cs

```diff
@@ -174,6 +168,16 @@ private async Task SendWelcomeMessageAsync(ITurnContext turnContext, Cancellatio
             }
         }
 
+        /// <summary>
+        /// Sends a rank request to Personalizer using randomly selected weather/date as the context features
+        /// and the available coffees and teas as the actions
+        /// </summary>
+        /// <param name="turnContext">A <see cref="ITurnContext"/> containing all the data needed
+        /// for processing this conversation turn.</param>
+        /// <param name="eventId">EventId for this particular rank request</param>
+        /// <param name="cancellationToken">Optional) A <see cref="CancellationToken"/> that can be used by other objects
+        /// or threads to receive notice of cancellation.</param>
+        /// <returns>A <see cref="Task"/> representing the rank response from Personalizer.</returns>
         private async Task<RankResponse> ChooseRankAsync(ITurnContext turnContext, string eventId, CancellationToken cancellationToken)
         {
             IList<object> contextFeature = new List<object>
```

Comment:
Line 225 - please add comment that highlights this is one of the 2 calls to Personalizer.

Reply:
added comment